debian/docker/Dockerfile

```diff
@@ -86,6 +80,12 @@ RUN wget https://deb.nodesource.com/setup_${NODEJS_VERSION}.x -O nodejs-install.
  && apt-get install -y \
     nodejs
 
+# Clean up dependencies
```

Comment:
Same as above.

I'm OK with moving it to its own layer, but I was under the impression that cleaning up within each later is better. Any Docker docs on this would be appreciated.

Reply:
Should I keep the clean up in every layer then ?